components/bl-signature-pad-component/src/buttons/undo-button.js

```diff
@@ -0,0 +1,26 @@
+export default function UndoButton(props) {
+  const { signaturePad, component, eventHandlers } = props;
+
+  const onUndoClick = () => {
+    const data = signaturePad.toData();
+
+    if (data) {
+      data.pop();
+      signaturePad.fromData(data);
+    }
+
+    if (eventHandlers.onUndoClick) {
+      eventHandlers.onUndoClick();
+    }
+  };
+
+  return (
+    <button
```

Comment:
Could we add classes and variables for all buttons to style them separately? Current behavior should remain too.